examples/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+function(rwkv_add_example source)
```

Comment:
As you yourself said,

>"Examples" is just a blanket term for "executables that use the library, that are provided with the distribution".

I agree, and would prefer more generic, but more correct term like "scripts".